nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -275,16 +275,23 @@ let
         redirectListen = filter (x: !x.ssl) defaultListen;
 
         acmeLocation = optionalString (vhost.enableACME || vhost.useACMEHost != null) ''
+          # Hide /acme-challenge subdirectory and return 404 on all requests.
+          # It is somewhat more secure than letting Nginx return 403.
```

Comment:
Please put the reason for why this is more secure.

Reply:
This is comment is quoted / taken verbatim from the source linked in the PR description.

Reply:
I've added a comment link to the source for anyone who wants to investigate

Reply:
This is not more secure, it just hides more that the acme-challenge is used

Reply:
That's a good characterization.  I'll update the comment to reflect this.

Reply:
Great, thank you!